transformer/app.js

```diff
@@ -195,29 +196,65 @@ const sendUpdateCommand = async (updateBody) => {
 const putFindingsIntos3bucket = async (failedFindings) => {
   const bucketParams = {
     Bucket: process.env.S3_BUCKET_NAME,
-    Key: `Failed_finding_${new Date().getTime().toString()}.json`,
+    Key: `failed_finding_${new Date().getTime().toString()}.json`,
     Body: JSON.stringify(failedFindings),
   };
   let data;
   try {
     data = await s3client.send(new PutObjectCommand(bucketParams));
-    logger.info(`Failed findings uploaded to s3://${bucketParams.bucketParams.Bucket}/${bucketParams.bucketParams.Key}`);
+    logger.info(`Failed findings uploaded to s3://${bucketParams.Bucket}/${bucketParams.Key}`);
   } catch (err) {
     logger.debug('Error while uploading failed findings to s3 bucket', { err });
   }
   return data;
 };
 
+const axiosClient = axios.create({
+  baseURL: 'https://heapanalytics.com/api',
+  headers: {
+    'Content-Type': 'application/json',
+    accept: 'application/json',
+  }
+});
+
+const sendCallHomeData = async (callHomePayload) => {
+  const APP_HEAPIO_APP_ID = process.env.APP_HEAPIO_APP_ID
+  let response;
+  if (!APP_HEAPIO_APP_ID) {
+    logger.warn('Missing APP_HEAPIO_APP_ID env var. No data sent.');
+    return;
+  }
+
+  try {
+    const body = {
+      app_id: APP_HEAPIO_APP_ID,
+      identity: callHomePayload.jpd_url,
+      event: 'transform-issue-and-send-to-security-hub',
+      properties: callHomePayload,
+    };
+
+    logger.info('Sending data to Heap.io, path: /track', { body });
+
+    response = await axiosClient.post('/track', body);
+  } catch (e) {
+    logger.error('Failed to send data to Heap.io', { error: e.toJSON() });
+  }
+  return response;
+}
+
+
 export async function lambdaHandler(event, context) {
   let response;
+  let issue;
```

Comment:
Let's initialize this to empty object, to be on safe side.

Reply:
Ok.